tests/normalization_pvwatts_test.py

```diff
@@ -40,7 +40,9 @@ def setUp(self):
         energy_index = pd.date_range(start='2012-01-01',
                                      periods=12,
                                      freq='MS')
-        dummy_energy = np.repeat(a=19.75, repeats=12)
+        power_meas = 19.75  # power in dummy conditions
+        hours = (energy_index - energy_index.shift(-1)).astype('int64') / (10.0**9 * 3600.0)
```

Comment:
I think pandas datetimeIndexes are phasing out use of the minus operator in newer versions, and instead include a `.difference()` method. But, this unit test will fail when that happens!